service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/OrganizationMapper.java

```diff
@@ -29,6 +32,20 @@
 
     private final NodeUtil nodeUtil;
 
+    public Organization mapOrganization(POCDMT000002UK01InformationRecipient informationRecipient) {
+        POCDMT000002UK01IntendedRecipient intendedRecipient = informationRecipient.getIntendedRecipient();
+        if (intendedRecipient.isSetReceivedOrganization()) {
+            Organization fhirOrganization = mapOrganization(intendedRecipient.getReceivedOrganization());
+            Coding code = new Coding();
```

Comment:
You could join setting to new Coding() and in other `mapOrganization` method there's nicer way to get Organization name:
```suggestion
            Coding code = new Coding()
                .setCode(String.valueOf(informationRecipient.getTypeCode()))
             .setDisplay(nodeUtil.getNodeValueString(intendedRecipient.getReceivedOrganization().getNameArray(0)));
```
Then you'll also have to update the expectedResult.json.